cmd/mt-kafka-mdm-report-out-of-order/applicationflags.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type ApplicationFlags struct {
+	flagSet *flag.FlagSet
+
+	runDurationStr   string
+	ConfFile         string
+	PartitionFrom    int
+	PartitionTo      int
+	graceDurationStr string
+	Prefix           string
+	Substr           string
+	DoUnknownMP      bool
+	GroupByName      bool
+	GroupByTag       string
+
+	// after parsing
+	RunDuration   time.Duration
+	GraceDuration time.Duration
```

Comment:
This could probably use a better name, maybe "grace period" or something more closely related to the reorder buffer?